### ISSUE TITLE
Issue with log4js.shutdown

### ISSUE THREAD

**Issue by konstantin-kornienko-epam** [log4js-node/rabbitmq#2]
One will got an error when trying to shutdown log4js (f.e. before clean exit):

```
var log4js = require('log4js');
log4js.configure({
  appenders: {
    mq: {
      type: '@log4js-node/rabbitmq',
      host: 'localhost',
      port: 5672,
      username: 'guest',
      password: 'guest',
      routing_key: 'test',
      exchange: 'logstash',
      mq_type: 'direct'
    }
  },
  categories: { default: { appenders: ['mq'], level: 'info' } }
});

var logger = log4js.getLogger();
logger.info("OK");
log4js.shutdown(); // HERE error will thrown
process.exit();
```
error is:

```
        client.close().then(done);
               ^

TypeError: client.close is not a function
    at Function.appender.shutdown (C:\logs-to-amqp-test\log4js\node_modules\@log4js-node\rabbitmq\lib\index.js:48:9)
    at appendersToCheck.filter.forEach.a (C:\logs-to-amqp-test\log4js\node_modules\log4js\lib\log4js.js:105:59)
...
```

'client' is a Promise, it have not 'close' method. Dont' know how it should be fixed :(

**Comment by nomiddlename** [comment]
This should be fixed in version 1.1.0. Please re-open the issue if it isn't. Thanks.

**Comment by konstantin-kornienko-epam** [comment]
Now shutdown is performing silently ;). Thank you! :)
